packages/web3-providers-ws/src/index.ts

```diff
@@ -0,0 +1,543 @@
+import { request, w3cwebsocket } from 'websocket';
```

Comment:
```suggestion
import { w3cwebsocket } from 'websocket';
```

Seems to be an unused import